src/client/opamInitDefaults.ml

```diff
@@ -32,20 +32,73 @@ let eval_variables = [
   "OCaml version present on your system independently of opam, if any";
 ]
 
```

Comment:
There is quite a bit of evaluation below: it'd be best to make `init_config` (and its dependencies) a function of `unit`